torch/nn/parallel/scatter_gather.py

```diff
@@ -80,10 +76,7 @@ def scatter_kwargs(
 
 
 def gather(outputs: Any, target_device: Union[int, torch.device], dim: int = 0) -> Any:
-    r"""
-    Gathers tensors from different GPUs on a specified device.
-    Use 'cpu' for CPU to avoid a deprecation warning.
-    """
+    r"""Gather tensors from different GPUs on a specified device. Use 'cpu' for CPU to avoid a deprecation warning."""
```

Comment:
```suggestion
    r"""Gather tensors from different GPUs on a specified device. 
    
    Use 'cpu' for CPU to avoid a deprecation warning."""
```